src/main/java/com/twilio/oai/DirectoryStructureService.java

```diff
@@ -180,6 +185,21 @@ private List<String> fetchNonParentPathParams(Operation operation){
         return null;
     }
 
+    private List<Parameter> fetchNonParentPathParams(Operation operation){
+        List<Parameter> params = new ArrayList<>();
+        if (null == operation) return params;
+        List<Parameter> pathParams = Optional.ofNullable(operation.getParameters())
+                .stream().flatMap(Collection::stream)
+                .filter(param -> Objects.nonNull(param.getIn())).filter(PathUtils::isPathParam)
+                .collect(Collectors.toList());
+        params = pathParams.stream().filter(parameter -> Objects.isNull(parameter.getExtensions()))
+                .collect(Collectors.toList());
+        params.addAll(pathParams.stream().filter(parameter -> Objects.nonNull(parameter.getExtensions()))
+                    .filter(parameter -> !PathUtils.isParentParam(parameter))
+                    .collect(Collectors.toList()));
+        return params;
```

Comment:
This looks almost identical to `fetchNonParentPathParamNames`. Looks like you can refactor so one calls the other.